src/Honeybee_Honeybee.py

```diff
@@ -5022,7 +5025,116 @@ def assignScheduleBasedOnProgram(self, component = None):
         
         # find all the patameters and assign them to 
         self.isSchedulesAssigned = True
+        
+    def getConstructionsByStandardClimateZone(self,standard,climateZone,component=None):
+        
+        openStudioStandardLib = sc.sticky["honeybee_OpenStudioStandardsFile"]
+        
+        try:
+            schedulesAndLoads = openStudioStandardLib['space_types']['90.1-2007']['ClimateZone 1-8'][self.bldgProgram][self.zoneProgram]
+            
+        except:
```

Comment:
Are you expecting a `keyError`? I would break this down into two steps so you can return a clear error message.

```python
    try:
        schedulesAndLoadsforProgram = openStudioStandardLib['space_types']['90.1-2007']['ClimateZone 1-8'][self.bldgProgram]
   except keyError:
       # now you are sure the issue is with the program
       print '...'
   else:
      try:
          schedulesAndLoads = schedulesAndLoadsforProgram[self.zoneProgram]
      except keyError:
          ....

```